source/Calamari/Commands/RunScriptCommand.cs

```diff
@@ -50,6 +51,7 @@ ISubstituteInFiles substituteInFiles
             Options.Add("package=", "Path to the package to extract that contains the script.", v => packageFile = Path.GetFullPath(v));
             Options.Add("script=", $"Path to the script to execute. If --package is used, it can be a script inside the package.", v => scriptFileArg = v);
             Options.Add("scriptParameters=", $"Parameters to pass to the script.", v => scriptParametersArg = v);
+            Options.Add("timeout=", $"Timeout for script in milliseconds.", v => timeout = int.TryParse(v, out var timeoutMs) && timeoutMs > 0 ? TimeSpan.FromMilliseconds(timeoutMs) : TimeSpan.Zero);
```

Comment:
This isn't needed. The timeout will should always be set via the variable bag and `SpecialVariables.Action.Script.Timeout`